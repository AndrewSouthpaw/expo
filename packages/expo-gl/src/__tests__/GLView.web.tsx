import React from 'react';
import renderer from 'react-test-renderer';
import GLView from '../GLView.web';

it('renders correctly', () => {
  const tree = renderer.create(<GLView onContextCreate={() => {}} />);
  expect(tree).toMatchSnapshot();
});
