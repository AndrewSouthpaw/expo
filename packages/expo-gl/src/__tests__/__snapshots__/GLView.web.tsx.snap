// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "flex": 1,
    }
  }
>
  <canvas
    height={0}
    style={
      Object {
        "flex": 1,
        "height": 0,
        "width": 0,
      }
    }
    width={0}
  />
</div>
`;
